web_src/js/features/repo-unicode-escape.js

```diff
@@ -0,0 +1,28 @@
+export function initEscapeButton() {
+  $(document, 'a.escape-button').on('click', (e) => {
```

Comment:
Did you mean `$(document).on('click', 'a.escape-button', (e) => {}`?

Reply:
oops.

Reply:
```suggestion
  $(document).on('click', 'a.escape-button', (e) => {
```